firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -203,53 +215,20 @@ public static FirebaseFirestore testFirestore() {
    * Initializes a new Firestore instance that uses the default project, customized with the
    * provided settings.
    */
-  public static FirebaseFirestore testFirestore(FirebaseFirestoreSettings settings) {
-    FirebaseFirestore firestore = testFirestore(provider.projectId(), Level.DEBUG, settings);
-    primeBackend();
-    return firestore;
-  }
-
-  private static void primeBackend() {
```

Comment:
This only serves to slow down tests. Also, the test runner will actually run this code many times, because tests are run in separate process. This doesn't prevent timeout, because @Before is part of timeout.